src/main/scala/is/hail/asm4s/package.scala

```diff
@@ -119,6 +122,18 @@ package object asm4s {
     def newArray() = new IntInsnNode(NEWARRAY, T_CHAR)
   }
 
+  implicit object UnitInfo extends TypeInfo[Unit] {
+    val name = "V"
+    def loadOp = ???
+    def storeOp = ???
+    def aloadOp = ???
+    def astoreOp = ???
+    val returnOp = RETURN
+    override val slots = 2
+
+    def newArray() = new IntInsnNode(NEWARRAY, T_CHAR)
```

Comment:
Hmm ... ???  You can't create an array of void in Java.

```
class Foo {
    void main(String[] args) {
        void[] a = new void[10];
    }
}

foo.java:3: error: illegal start of expression
        void[] a = new void[10];
        ^
```

Same code with compiles fine.